content/swiftui/concepts/gestures/gestures.md

```diff
@@ -13,126 +13,116 @@ CatalogContent:
   - 'path/iOS-path'
 ---
 
-**Gestures** are used to make the view respond to various physical touches like taps, swipes, drags, and other standard screen motions.
+**Gestures** are used to make views respond to various physical touches like taps, swipes, drags, and other standard screen motions. (Modifiers)[https://www.codecademy.com/resources/docs/swiftui/viewmodifier] handle the logic behind gestures like a tap.
 
 ## Tap Gestures
 
-This is the most basic type of gesture. The `.onTapGesture()` modifier can be called on the view desired to track the gesture on:
+Call the `.onTapGesture()` modifier on a view to track the tap gesture on:
 
-```Swift
-struct Tap: View {
-    var body: some View {
-        Text("Hello, World!")
-            .onTapGesture {
-                print("View Tapped")
-            }
-    }
+```swift
+var body: some View {
+    Text("Hello, World!")
+        .onTapGesture {
+            print("View Tapped")
+        }
 }
 ```
 
-The output looks like this:
-
-![This is a gif that shows the effect of using TapGesture in Xcode simulation](https://raw.githubusercontent.com/Codecademy/docs/main/media/swiftui-tapgesture-example.gif)
+This will display:
 
-Every time the text view `"Hello, World"` is tapped, it prints `"View Tapped"` in the console.
+![TapGesture in Xcode Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/swiftui-gesture-tapgesture.gif)
 
-The action can also be implemented after a certain amount of tap gestures. For example:
+Every time the view is tapped, the code block within the `.onTapGesture()` modifier runs. This modifier can also accept an integer argument named `count`:
 
-```Swift
-struct Tap: View {
-    var body: some View {
-        Text("Hello, World!")
-            .onTapGesture(count: 3) {
-                print("View Tapped")
-            }
-    }
+```pseudo
+.onTapGesture(count: number) {
+    Code block
 }
 ```
 
-In this example, it prints `"View Tapped"` after taping the text view three times.
+After tapping the text view some `number` of times, the code block will run. For example, if `number` was set to `3`, three taps would cause the code block to be run.
 
-> **Note:** When a button is needed, a `Button` instance should be used rather than a tap gesture. Any view can be used as the button’s label, and the button type automatically provides many of the standard behaviors that users expect, like accessibility labels and hints.
+> **Note:** For buttons, create a (`Button`)[https://www.codecademy.com/resources/docs/swiftui/views/button] instance rather than a tap gesture. Buttons automatically provide standard behaviors like accessibility labels and hints.
```

Comment:
```suggestion
> **Note:** For buttons, create a [`Button`](https://www.codecademy.com/resources/docs/swiftui/views/button) instance rather than a tap gesture. Buttons automatically provide standard behaviors like accessibility labels and hints.
```